dc.dev.yml

```diff
@@ -23,6 +23,8 @@ services:
             - "${OPENSLIDES_DATASTORE_WRITER_PORT}:${OPENSLIDES_DATASTORE_WRITER_PORT}"
         environment:
             - DATASTORE_ENABLE_DEV_ENVIRONMENT=1
+            - COMMAND=create_initial_data
```

Comment:
This should be moved into the entrypoint: If `DATASTORE_INITIAL_DATA_FILE` is given, execute the create_initial_data script. Maybe with the env var as a parameter, so you do not need the env-service.

Reply:
I left it this way since it still leaves the possibility for multiple commands open. If we're sure we don't need any more or we want to implement them all via an env var, I can change this